hack/jenkins/common.ps1

```diff
@@ -41,17 +41,26 @@ gsutil.cmd -m cp -r gs://minikube-builds/$env:MINIKUBE_LOCATION/windows_cleanup_
 gsutil.cmd -m cp -r gs://minikube-builds/$env:MINIKUBE_LOCATION/windows_cleanup_cron.ps1 out/
 ./out/windows_cleanup_cron.ps1
 
-# Make sure Docker is up and running
-gsutil.cmd -m cp -r gs://minikube-builds/$env:MINIKUBE_LOCATION/setup_docker_desktop_windows.ps1 out/
-./out/setup_docker_desktop_windows.ps1
-If ($lastexitcode -gt 0) {
-	echo "Docker failed to start, exiting."
-
-	$json = "{`"state`": `"failure`", `"description`": `"Jenkins: docker failed to start`", `"target_url`": `"https://storage.googleapis.com/$gcs_bucket/Hyper-V_Windows.txt`", `"context`": `"$env:JOB_NAME`"}"
+# check for Docker CLI
+docker --help
```

Comment:
I like this ! i think there are placed in actual minikube code we use docker version to check if docker cli is there... (i think) and we could use this trick over there too